test/business/suggestionServiceTest.js

```diff
@@ -80,6 +78,18 @@ describe('Suggestion Service', () => {
       sort: 'releaseDate'
     })
   })
+
+  it('will include \'discovered\' licenses for declared license suggestions', async () => {
+    const t2 = EntityCoordinates.fromString('gem/rubygems/-/autobuild/1.6.2.b8')
+    const sample_definition = require('./evidence/issue-453-sample-1.json')
+    const service = setup(sample_definition, [])
+    const suggestions = await service.get(t2)
+    expect(suggestions).to.not.be.null
+    const declared = get(suggestions, 'licensed.declared')
+    expect(declared).to.equalInAnyOrder([
+      { value: 'GPL', version: '2.0' }
```

Comment:
see comment above about license version.

this should be `{ value: 'GPL-2.0', version: '1.6.2.b8' }`

Reply:
Yes, got it.

Reply:
Incorporated at https://github.com/clearlydefined/service/pull/507/commits/84a2dfb19418482f183eeff276d4aa3478004600